### PR TITLE
feat(docs): add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+Wrangler is an open source project because we believe that folks should have access, insight,
+and the opportunity to contirbute to their developer tools. Wrangler is also a product
```
Comment (xtuc) [inline CONTRIBUTING.md]:
```suggestion
and the opportunity to contribute to their developer tools. Wrangler is also a product
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## People
+
+Wrangler is maintained by @ashleygwilliams, and her team, [Workers Developer Experience].
+
+[Workers Developer Experience]: https://github.com/orgs/cloudflare/teams/workers-devexp
+
+## Primary Issue Triage
+
+Within 3 days, any incoming issue should be triaged. Triage involves:
```
Comment (xtuc) [inline CONTRIBUTING.md]: > Within 3 days

I wouldn't commit on that

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+- reading the issue and requesting any further information
+- always thank the user for submitting
```
Comment (xtuc) [inline CONTRIBUTING.md]: I doubt I (we) will do it for every issues.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+### Labelling
+
+- label all issues coming from non-team members with User Report
```
Comment (xtuc) [inline CONTRIBUTING.md]: Could we add it to the template? and ask member to remove the label.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- labelling the category of the issue: Feature, External Bug, Bug, Maintenance, Docs
+- optionally labelling a secondary category: Webpack, Routes, Workers Runtime, Refactor
+- labelling the status of the issue: Need More Info, Needs Repro, Needs Design, PR Welcome
+- optionally labelling other calls to action: Help Wanted, Question
+
+### Assignment
+
+- if the issue will require a large amount of back and forth between the reporter and the team
```
Comment (xtuc) [inline CONTRIBUTING.md]: I like that, maybe during our weeklys we could assign as much issues as possible to keep things moving and scoped.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    assign a single team member to manage the conversation
+
+## Product Issue Triage
+
+Once a week, the team meets to do Product Triage. This is where we assign work and update
+our plans for the milestones and releases.
+
+### Labelling
+
+- labelling the priority of the issue: Critical, Nice to Have, Low Priority
```
Comment (xtuc) [inline CONTRIBUTING.md]: I think this should be done in a meeting, priorities can be relative; we should all agree.

Comment (ashleygwilliams) [reply CONTRIBUTING.md]: this is done in a meeting- it says in the prose under the title ;)

File: CONTRIBUTING.md (hunk 3 above)
Comment (ashleygwilliams) [reply CONTRIBUTING.md]: we will. it's something i would like to enforce :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (ashleygwilliams) [reply CONTRIBUTING.md]: we are going to be committing to this as a goal, though i'll add a thing for holiday periods

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+Within 3 days, any incoming issue should be triaged. Triage involves:
+
+- reading the issue and requesting any further information
+- always thank the user for submitting
+
+### Labelling
+
+- label all issues coming from non-team members with User Report
+- labelling the category of the issue: Feature, External Bug, Bug, Maintenance, Docs
+- optionally labelling a secondary category: Webpack, Routes, Workers Runtime, Refactor
```
Comment (xtuc) [inline CONTRIBUTING.md]:
```suggestion
- optionally labelling a secondary category: webpack, Routes, Workers Runtime, Refactor
```

According to their branding

**Comment by granjef3** [comment]
This might be too much of a maintenance burden, but I think it would be nice if we had a small section on "How to work on wrangler" with like a very high-level map of the codebase, and how to build/install a local copy of wrangler.

**Comment by ashleygwilliams** [comment]
@granjef3 i'm super down for that! if you open an issue i think that'd be great

File: CONTRIBUTING.md (hunk 4 above)
Comment (ashleygwilliams) [reply CONTRIBUTING.md]: #250

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+### Labelling
+
+- All work-in-progress PRs should be labelled Work In Progress and the title should be
+    annotated [WIP] for easy scanning. No WIP PRs will be reviewed until the annotations
+    are removed.
+- All PRs that need to be reviewed should be labelled Needs Review until they have 
+    received all required reviews.
+- All PRs should be labelled with a changelog label: BREAKING, Feature, Bug, Maintenance, Docs
+
+### Reviewing
```
Comment (EverlastingBugstopper) [inline CONTRIBUTING.md]: Is this section TODO or should we remove the heading?

**Pull request merged**